lib/cell.js

```diff
@@ -0,0 +1,21 @@
+module.exports = class Cell {
+  constructor() {
+    this.symbol = null;
+  }
+
+  getSymbol() {
+    return this.symbol;
+  }
+
+  markWithSymbol(symbol) {
+    this.symbol = symbol;
+  }
+
+  isAvailable() {
+    return this.symbol === null;
+  }
+
+  isTaken() {
```

Comment:
I like the intention. I would be more likely to only have `isTaken` or `isAvailable`, the reason behind that is to avoid to lately have that:
```
isTake()
!isAvailable()
isAvailable()
!isTaken()
```

Here the project is small anough to avoid that but could happen in a larger code base